src/Umbraco.Infrastructure/Persistence/Dtos/PropertyDataDto.cs

```diff
@@ -23,7 +23,7 @@ internal class PropertyDataDto
 
     [Column("versionId")]
     [ForeignKey(typeof(ContentVersionDto))]
-    [Index(IndexTypes.UniqueNonClustered, Name = "IX_" + TableName + "_VersionId", ForColumns = "versionId,propertyTypeId,languageId,segment")]
+    [Index(IndexTypes.NonClustered, Name = "IX_" + TableName + "_VersionId", ForColumns = "versionId,propertyTypeId,languageId,segment", IncludeColumns = "varcharValue")]
```

Comment:
This would need to remain a unique index.

Reply:
Adding the `varcharValue` will significantly increase the index size and time to insert/update property data: why is this needed?

Reply:
TIL: I see it's added as included column, so it's not included in the index key columns (see [Create indexes with included columns](https://learn.microsoft.com/en-us/sql/relational-databases/indexes/create-indexes-with-included-columns?view=sql-server-ver16)).

Reply:
Perhaps revert this one line for now @ronaldbarendse ?

Reply:
Just trying to resurrect this PR. @ronaldbarendse @nzdev, can you confirm if this line should be reverted?